src/pages/LandingPage.tsx

```diff
@@ -78,22 +90,50 @@ export const LandingPage: FunctionComponent = () => {
 
             <PageSection variant={PageSectionVariants.light} className="pf-u-p-md">
               <div className="pf-u-text-align-right">
-                <Button variant="link" icon={<ThIcon />} className="pf-u-mr-sm" isInline isLarge/>
-                <Button variant="link" icon={<ThListIcon />} isInline isLarge isDisabled/>
+                <Button variant="link" icon={<ThIcon />} onClick={() => setView('grid')} className="pf-u-mr-sm" isInline isLarge/>
+                <Button variant="link" icon={<ThListIcon />} onClick={() => setView('list')} isInline isLarge/>
               </div>
             </PageSection>
 
             <PageSection className="apid-c-page__main-section-gallery" style={paginatedGalleryInfo.height ? {minHeight: `${paginatedGalleryInfo.height}px !important`} : undefined} padding={{ default: 'noPadding' }} isFilled={true}>
-              <GalleryTemplate
-                  id={galleryId}
-                  elements={filteredDocs}
-                  isHidden
-              />
-              { paginatedGalleryInfo.paginatedElements.length > 0 ?
+            { view === 'grid'
```

Comment:
These ternary operations are confusing - I would group each of these into a component (either on the same file):
```
{ view === 'grid' ? 
<GridContent {/* all props required for Grid */} /> : 
<TableContent {/* all props required for table */} /> }
```

or conditionally include each:

```
{ view === 'grid' && <>
   <GalleryTemplate ... // etc
</>}

{ view === 'list' && <>
   <Table ... //etc
</>
```

Note that this can be a follow up on this PR

Reply:
I can do that in a follow up for sure :smile: